config.yml

```diff
@@ -37,6 +37,9 @@ redis_activity_consumers: 8
 # Number of concurrent connections to Redis in the connection pool.
 redis_connection_pool_size: 20
 
+# Max number of seconds BLPOP waits before returning anyway. Must be a positive integer. 0 means wait indefinitely.
+redis_blpop_timeout: 10
```

Comment:
because I wanted a "safe" value for all the possible environments, I'm not sure that zero has no side effects. Anyway the change is trivial so let me know if I have to change it to zero.